mapie_v1/integration_tests/tests/test_regression.py

```diff
@@ -195,56 +184,148 @@ def test_intervals_and_predictions_exact_equality_cross(params_cross):
+            "cv": Subsample(n_resamplings=3,
+                            replace=True,
+                            random_state=RANDOM_STATE),
+            "method": "plus",
+            "optimize_beta": True,
+            "random_state": RANDOM_STATE,
+        },
+        "v1": {
+            "estimator": positive_predictor,
+            "confidence_level": [0.5, 0.5],
+            "aggregation_method": "mean",
+            "conformity_score": "gamma",
+            "resampling": Subsample(n_resamplings=3,
+                                    replace=True,
+                                    random_state=RANDOM_STATE),
+            "method": "plus",
+            "minimize_interval_width": True,
+            "random_state": RANDOM_STATE,
+        },
+    }
+]
 
-    v0_preds, v0_pred_intervals = v0.predict(X_cross, **v0_predict_params)
 
-    v1_pred_intervals = v1.predict_set(X_cross, **v1_predict_set_params)
-    if v1_pred_intervals.ndim == 2:
-        v1_pred_intervals = np.expand_dims(v1_pred_intervals, axis=2)
-    v1_preds = v1.predict(X_cross, **v1_predict_params)
+@pytest.mark.parametrize("params_jackknife", params_test_cases_jackknife)
+def test_jackknife_after_bootstrap_consistency(params_jackknife):
+    v0_params = params_jackknife["v0"]
+    v1_params = params_jackknife["v1"]
 
-    np.testing.assert_array_equal(v0_preds, v1_preds)
-    np.testing.assert_array_equal(v0_pred_intervals, v1_pred_intervals)
+    v0, v1 = initialize_models("jackknife", v0_params, v1_params)
+    evaluate_models(v0, v1, X, y, v0_params, v1_params)
 
 
 def initialize_models(
     strategy_key,
     v0_params: dict,
     v1_params: dict,
 ):
-    if strategy_key == "prefit":
-        v0_params.update({"cv": "prefit"})
-        v0_params = filter_params(MapieRegressorV0.__init__, v0_params)
-        v1_params = filter_params(SplitConformalRegressor.__init__, v1_params)
-        v0 = MapieRegressorV0(**v0_params)
-        v1 = SplitConformalRegressor(prefit=True, **v1_params)
-
-    elif strategy_key == "split":
-        v0_params.update({"cv": "split"})
-        v0_params = filter_params(MapieRegressorV0.__init__, v0_params)
+    if strategy_key == "split":
         v1_params = filter_params(SplitConformalRegressor.__init__, v1_params)
-        v0 = MapieRegressorV0(**v0_params)
         v1 = SplitConformalRegressor(**v1_params)
 
+    elif strategy_key == "cross":
+        v1_params = filter_params(CrossConformalRegressor.__init__, v1_params)
+        v1 = CrossConformalRegressor(**v1_params)
+
+    elif strategy_key == "jackknife":
+        v1_params = filter_params(
+            JackknifeAfterBootstrapRegressor.__init__,
+            v1_params
+        )
+        v1 = JackknifeAfterBootstrapRegressor(**v1_params)
+
     else:
         raise ValueError(f"Unknown strategy key: {strategy_key}")
 
+    v0_params = filter_params(MapieRegressorV0.__init__, v0_params)
+    v0 = MapieRegressorV0(**v0_params)
+
     return v0, v1
+
+
+def evaluate_models(
```

Comment:
`evaluate_models` is a bit of a vague name

Reply:
replacer by compare_model_predictions_and_intervals